WtWSMS/common/culture/creation_names/BP_ck2_custom_names.txt

```diff
@@ -0,0 +1,264 @@
+#Note that since these *aren't* county events in CK3, we don't do the religion checks wrt counties
+#ROM.1000: Skipped (Duplicate French Creation)
+#ROM.1100: Skipped (Duplicate Occitan Creation)
+#ROM.1150: Skipped (Visigoth back to Romano-Gallic)
+#ROM.2000: Skipped (Glorious Gothi)
+#ROM.2001: Skipped (Continential Celtic Province to Romano-Hispanic)
+#ROM.2048: Skipped (Gaulish Province to Romano-Gallic)
+#ROM.2100: Skipped (Iberian Clusterfuck Part 1: Catalans)
+#ROM.2200: Skipped (Iberian Clusterfuck Part 2: Portuguese)
+#ROM.2300: Skipped (Iberian Clusterfuck Part 3: Castilian)
+#ROM.2400: Skipped (Iberian Clusterfuck Part 4: Leonese)
+#ROM.2500: Skipped (Iberian Clusterfuck Part 5: Andalusian)
+#ROM.3000: Skipped (Latin to Italian)
+#CELT.1000: Skipped (Pictish to Scottish)
+breton = { #CELT.1100: Amorican to Breton
+	trigger = {
+		root = {
+			OR = {
+				culture = culture:briton
+				culture = culture:romano_british
+			}
+			capital_county = { any_county_province = { geographical_region = custom_breton } }
+		}
+		scope:other_culture = culture:armorican
+	}
+	hybrid = yes
+}
+##### THIS SECTION BASED ON CK2/WtWSMS/events/WtWSMS_melting_pot_evnts_events.txt
+#Note that since these *aren't* county events in CK3, we don't do the religion checks wrt counties
+italian = { #WtWSMS_pots.35; Roman to Italian
+	trigger = {
+		root = {
+			romance_hybrid_ruler_valid_trigger = {
+				CULTURE_TRIGGER = italian_ruler_culture_trigger
+				CULTURE_REGION = world_europe_south_italy
+				REGION_LOCKED = yes
+			}
+		}
+		scope:other_culture = culture:roman
+	}
+}
+frankish = { #WtWSMS_pots.41; Romano-Gallic to Old French
+	trigger = {
+		root = {
+			romance_hybrid_ruler_valid_trigger = {
+				CULTURE_TRIGGER = frankish_culture_trigger
+				CULTURE_REGION = custom_frankish
+				REGION_LOCKED = yes
+			}
+		}
+		scope:other_culture = culture:romano_gallic
+	}
+	hybrid = yes
+}
+gallo_romance = { #WtWSMS_pots.41; Romano-Gallic to Gallo-Romance
+	trigger = {
+		root = {
+			romance_hybrid_ruler_valid_trigger = {
+				CULTURE_TRIGGER = gallo_romance_culture_trigger
+				CULTURE_REGION = custom_frankish
+				REGION_LOCKED = yes
+			}
+		}
+		scope:other_culture = culture:romano_gallic
+	}
+	hybrid = yes
+}
+occitan = { #WtWSMS_pots.51; Romano-Gallic to Occitan
+	trigger = {
+		root = {
+			romance_hybrid_ruler_valid_trigger = {
+				CULTURE_TRIGGER = occitan_culture_trigger
+				CULTURE_REGION = custom_occitan
+				REGION_LOCKED = yes
+			}
+		}
+		scope:other_culture = culture:romano_gallic
+	}
+	hybrid = yes
+}
+catalan = { #WtWSMS_pots.57; Romano-Hispanic to Catalan
+	trigger = {
+		root = {
+			romance_hybrid_ruler_valid_trigger = {
+				CULTURE_TRIGGER = occitan_culture_trigger
+				CULTURE_REGION = custom_catalan
+				REGION_LOCKED = yes
+			}
+		}
+		scope:other_culture = culture:romano_hispanic
+	}
+	hybrid = yes
+}
+#WtWSMS_pots.62 - Portuguese (Not ported for now)
```

Comment:
Any reason for why this culture in particular isn't ported? Not to add work but just to understand the underlying choice

Reply:
Portuguese has their own Vanilla decision, like English culture.

See: https://ck3.paradoxwikis.com/Decisions
Cross-Reference "Found Portugal"

Reply:
In my view relying on that doesn't make much sense given that the melting-pots are meant to cover the entire area, that decision corresponds to the 1066 setup, not the mod.

Reply:
Except Portuguese itself is more of a divergence from the earlier Galician culture/language. That' doesn't quite match up with history or WtWSMS_pots.62; WtWSMS_pots.62 pretty much "instantly" creates these cultures from Romano-Hispanic, whereas I think we'd have Galician evolve from Romano-Hispanic and Portuguese branch from Galician.